firebase-firestore/src/main/java/com/google/firebase/firestore/encoding/MapEncoder.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 Google LLC
```

Comment:
To prevent anything inside `/encoding/` of being included in the public API, you can use a `package-info` file and declare it as `@hide`. See https://github.com/firebase/firebase-android-sdk/blob/master/firebase-firestore/src/main/java/com/google/firebase/firestore/index/package-info.java

Reply:
Can I do the same thing with any Kotlin folder? like `/firebase/firestore/ktx/serialization/`
 Didn't find any example in this code base.